packages/lodestar/src/chain/forkChoice/statefulDag/lmdGhost.ts

```diff
@@ -242,7 +242,14 @@ export class StatefulDagLMDGHOST implements ILMDGHOST {
       this.nodes[parentRoot].addChild(node);
     }
     if (justifiedCheckpoint && (!this.justified || justifiedCheckpoint.epoch > this.justified.epoch)) {
-      this.checkAndSetJustified(justifiedCheckpoint);
```

Comment:
the logic is right but probably we want to do everything inside `checkAndSetJustified`

Reply:
You're talking about the `if` statements surrounding that call to `checkAndSetJustified`, correct?

Reply:
yes. I mean we keep the `this.checkAndSetJustified(justifiedCheckpoint);` call and change the logic inside `checkAndSetJustified` because I think it's better to centralize the check and assignment of `bestJustifiedCheckpoint` in one place

Reply:
so sorry, I misunderstood you. What I mean is we keep both the `if (justifiedCheckpoint && (!this.justified || justifiedCheckpoint.epoch > this.justified.epoch)) {` and `this.checkAndSetJustified(justifiedCheckpoint);` and just do the requested change inside `checkAndSetJustified`

Reply:
I'm not sure I understand. `checkAndSetJustified` is already doing the change, no?

Reply:
yeah it does its job, just want to keep minimal change to the current code but probably this looks good enough too. I may need to change the flow a little bit in my other PR anyway.